### ISSUE TITLE
Table view current sort indicator is clickable

### ISSUE THREAD

**Issue by michaelwood** [ThreeSixtyGiving/grantnav#901]
When in table view on `/search` the column which is currently being sorted on has a clickable sort icon. This is likely part of the progressive table loading library we use but it would be good to see if there is a work around for it at some point.

**Comment by Lathrisk** [comment]
@mariongalley I think that icon is baked into the library to indicate the which column is used for sorting and the sorting pattern has been applied. Noting that although the icon exists it isn't 'clickable' or functional in any way (despite the icon changing, another red herring). We might be able to remove this with some override CSS. Changing the libraries is another possible fix, depending on the library used.

**Comment by mariongalley** [comment]
@michaelwood Since you raised this one, what do you think?

**Comment by michaelwood** [comment]
yes likely the simplest way forward is to just override the css for this to display:none.

**Comment by michaelwood** [comment]
![image](https://github.com/user-attachments/assets/1015e4b3-9cea-4642-811a-2c0de29fb146)

https://grantnav.threesixtygiving.org/search?view_mode=table
